### DISCUSSION TITLE
我下载百度网盘的东西，但是下载完成后会在其他盘显示

### DISCUSSION THREAD

**Discussion by SmnmV** [agalwood/Motrix#1723] [Q&A]
设置了默认路径为C:user\xxx\download\baidunyun，下载完成之后会直接在D：的目录下，请问这是啥情况

**Comment by ApexGP** [comment]
> 设置了默认路径为C:user\xxx\download\baidunyun，下载完成之后会直接在D：的目录下，请问这是啥情况

这个好像是个bug，即使你设置了默认文件夹，`motrix` 倾向于在D盘创建下载文件夹，这好像是跟`aria2`的权限有关。我在一台只有C盘的笔记本上做过测试，客户端会直接返回错误码`18`，也就是无法创建路径`D:\Downloads`
